kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorIT.java

```diff
@@ -21,6 +22,7 @@
 import static io.confluent.connect.s3.S3SinkConnectorConfig.STORE_KAFKA_KEYS_CONFIG;
 import static io.confluent.connect.s3.S3SinkConnectorConfig.AWS_ACCESS_KEY_ID_CONFIG;
 import static io.confluent.connect.s3.S3SinkConnectorConfig.AWS_SECRET_ACCESS_KEY_CONFIG;
+import static io.confluent.connect.s3.continuum.S3ContinuumConfig.*;
```

Comment:
It looks like the [style guide](https://github.com/ni/kafka-connect-storage-cloud/blob/master/CONTRIBUTING.md#code-style) discourages importing *.

Reply:
Ah, I didn't realize there was a style guide!